spec/Dockerfile_spec.rb

```diff
@@ -146,14 +146,15 @@
               set_var(backend_type, :product, "PRODUCT_VAR", "product-var")
               set_var(backend_type, :service, "OLD_SERVICE_VAR", "old-service-var", old_keys: true)
               set_var(backend_type, :service, "SERVICE_VAR", "service-var")
+              set_var(backend_type, :peer, "PEER_ID", "peer-var", service_env: "peer-rise-runtime-1768")
```

Comment:
I don't think this is what you want .. you want to set something like `PEER_VAR` with a value of `peer-var` and asset that `PEER_VAR` gets set to a value of `peer-var` down below.

Reply:
`set_var(backend_type, :peer, "PEER_VAR", "peer-var", service_env: "peer-rise-runtime-1768")`

Reply:
And then in your tests:

`its(:stdout) { should include_env "PEER_VAR", "peer-var" }`